### DISCUSSION TITLE
Can't change 2FA phone number

### DISCUSSION THREAD

**Discussion by daveasdf** [community/community#102655] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

After reaching the 2FA form without any issues, the phone number field is greyed out, can't modify it and it's stuck on my old phone number. Have used up 2 recovery codes thus far, please advise.

**Comment by swesri** [comment]
I am able to edit the form,  however, I get the following error message -- 
"We tried delivering an SMS to that number, 
but we are not authorized to 
send SMS messages to this recipient"

**Comment by daveasdf** [comment]
Hello!  I sent this question to support, can you please explain how it got to a community forum and who you are smiley_face smiley_face smiley_face
